### PR TITLE
Add sorting users by creation timestamp

### PR DESCRIPTION
**Pull request by dklimpel**
Fixes: #172

### CODE REVIEW

File: src/components/users.js [hunk 1]
```diff
@@ -169,13 +197,35 @@ export const UserList = props => {
         />
         <TextField source="id" sortBy="name" />
         <TextField source="displayname" />
-        <BooleanField source="is_guest" />
-        <BooleanField source="admin" />
         <BooleanField source="deactivated" />
+        {adminFilter && <BooleanField source="admin" />}
+        {isGuestFilter && <BooleanField source="is_guest" />}
+        {creationTimeFilter && (
```
Comment (awesome-manuel) [inline src/components/users.js:203]: The columns should always be visible.

**Review by awesome-manuel — CHANGES_REQUESTED** [review]
I don't understand why the visibility switch for the column is added to the filters. This looks a bit wierd in the UI.

**Comment by dklimpel** [comment]
> I don't understand why the visibility switch for the column is added to the filters. This looks a bit wierd in the UI.

This is the same filter and behaviour like in the room and directory list. The idea is that the admin can customise the list view.

File: src/components/users.js (hunk 1 above)
Comment (dklimpel) [reply src/components/users.js:204]: Than the list is pretty wide if you do not have short user names.

**Comment by awesome-manuel** [comment]
> > I don't understand why the visibility switch for the column is added to the filters. This looks a bit wierd in the UI.
> 
> This is the same filter and behaviour like in the room and directory list. The idea is that the admin can customise the list view.

Right, but the name "Filter" is a bit strange, if it is used for enabling or disabling columns. There is a [<SelectColumnsButton>](https://marmelab.com/ra-enterprise/modules/ra-preferences#selectcolumnsbutton-store-datagrid-columns-in-preferences) in react-admin, but only for the paid version. Maybe we can implement a simple copy of that ourselfes.

**Comment by dklimpel** [comment]
I have no intention of doing that in this PR.

**Pull request merged**